code/client/src/constants/wallet.js

```diff
@@ -7,5 +7,9 @@ export default {
   maxTransferAttempts: 3,
   checkCommitInterval: 5000,
   fetchBalanceFrequency: 15000,
-  fetchDelaysAfterTransfer: [0, 2000, 5000]
+  fetchDelaysAfterTransfer: [0, 2000, 5000],
+  knownAddressKeys: {
```

Comment:
It is not necessary to separate them. In reality, I find I do a lot of transfer between recovery address and 1wallet addresses. Let's drop the keys and use a single array to store them.

Reply:
Removed this key separation.